software/impler.yml

```diff
@@ -0,0 +1,22 @@
+# software name
+name: "Impler"
+# URL of the software project's homepage
+website_url: "https://impler.io"
+# URL where the full source code of the program can be downloaded
+source_code_url: "https://github.com/implerhq/impler.io"
+# description of what the software does, shorter than 250 characters, sentence case
+description: "Data import experience, built for engineering teams to help them build rich data import experience without constantly reinventing the wheel."
+# list of license identifiers, see https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/licenses.yml for the full list of licenses
+licenses:
+  - MIT
+# list of languages/platforms, see https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/platforms for the full list of platforms
+platforms:
+  - Nodejs
+  - Docker
+# list of tags (categories), see https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/tags for the full list of tags
+tags:
+  - Software Development - Low Code
+# (optional, yes/no, default no) whether the software depends on a third-party service outside the user's control
+depends_3rdparty: no
+# (optional) link to an interactive demo of the software
+demo_url: "https://web.impler.io/auth/signin"
```

Comment:
Where can I find login credentials for the demo? If the demo requires signin, please remove it.

Reply:
Yes, users have to sign up to use the platform. Removed `demo_url`.